src/components/Form/FormUtils.js

```diff
@@ -257,6 +257,26 @@ export function arr_diff(a1, a2) {
     return diff;
 }
 
+// Function definition with passing two arrays
```

Comment:
You can make this faster by using .some & .includes 

```array1.some(item => array2.includes(item))``` 

The issue is ```.includes``` & ```.some``` isn't supported on IE.

Reply:
i knew there was a smarter way to do this xD. thanks!